src/mudlet.cpp

```diff
@@ -1984,7 +1984,7 @@ void mudlet::writeSettings()
     settings.setValue("enableFullScreenMode", mEnableFullScreenMode);
     settings.setValue("copyAsImageTimeout", mCopyAsImageTimeout);
     settings.setValue("interfaceLanguage", mInterfaceLanguage);
-    // value only used during PTBs, remove it to reduce confusion in the future
+    // 'darkTheme' value was only used during PTBs, remove it to reduce confusion in the future
```

Comment:
:confused: It could be helpful to indicate the timeframe (or by some other means) when this was the case...